tensorflow/lite/toco/tflite/import.cc

```diff
@@ -143,7 +143,7 @@ void ImportOperators(
 
     // Make sure all the inputs and outputs are hooked up.
     auto inputs = input_op->inputs();
-    for (int i = 0; i < inputs->Length(); i++) {
+    for (size_t i = 0; i < inputs->Length(); ++i) {
       auto input_index = inputs->Get(i);
```

Comment:
Won't you get a warning on 64-bit builds? The flatbuffer schema doesn't use size_t; it uses uffset_t, which is always uint32_t. Same elsewhere.

Reply:
no, we are using 64-bit build warning didn't appera.

Reply:
But for 64-bit builds, you're just going from an unsigned/signed conversion warning to a narrowing warning. What about using uint32_t instead of size_t?

Reply:
Yeah, with uint32_t using instead of size_t resolve warning and same commited.